src/vstart.sh

```diff
@@ -333,6 +334,9 @@ case $1 in
     --without-dashboard)
         with_mgr_dashboard=false
         ;;
+    -S )
```

Comment:
i'd suggest use "--with-spdk" instead. or use "--bluestore-spdk" to be more explicit on its dependency on bluestore.

Reply:
Thanks! I think "--bluestore-spdk" is better. Will update soon.